lab-alex-bessie-david/lib/transformer.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
```

Comment:
Lots of magic numbers in here. They obviously have some significance with respect to the buffer, but that significance is unknown when looking at the code. You should store those in descriptive variable names so someone looking at your code can look and understand what they stand for.